lisa/sut_orchestrator/azure/arm_template.json

```diff
@@ -275,6 +281,33 @@
                 "hyperVGeneration": "[concat('V', parameters('nodes')[copyIndex('imageCopy')]['hyperv_generation'])]"
             }
         },
+        {
+            "type": "Microsoft.Compute/disks",
+            "apiVersion": "2021-04-01",
+            "name": "[concat(parameters('nodes')[copyIndex('diskCopy')]['name'], '-disk')]",
+            "location": "[parameters('location')]",
+            "condition": "[not(empty(parameters('nodes')[copyIndex('diskCopy')]['vmgs']))]",
+            "copy": {
+                "name": "diskCopy",
+                "count": "[variables('node_count')]"
+            },
+            "sku": {
+                "name": "Standard_LRS"
+            },
+            "properties": {
+                "osType": "Linux",
+                "hyperVGeneration": "V2",
```

Comment:
This shouldn't be hard coded, even the cvm supports only v2.

Reply:
Resolved